docs/index.rst

```diff
@@ -40,140 +40,133 @@ contacting support.
+| :ref:`cassandra`                                 | >= 3.24       | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`cassandra`                                 | >= 3.5        | Yes            |
+| :ref:`celery`                                    | >= 4.4        | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`celery`                                    | >= 3.1        | Yes            |
+| :ref:`cherrypy`                                  | >= 17.0       | No             |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`cherrypy`                                  | >= 11.2.0     | No             |
+| :ref:`confluent-kafka <kafka>`                   | >= 1.9.2      | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`confluent-kafka <kafka>`                   | >= 1.7.0      | Yes            |
+| :ref:`consul`                                    | >= 1.1        | Yes [2]_       |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`consul`                                    | >= 0.7        | Yes [2]_       |
+| :ref:`django`                                    | >= 3.2        | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`django`                                    | >= 1.8        | Yes            |
+| :ref:`djangorestframework <djangorestframework>` | >= 3.11       | No             |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`djangorestframework <djangorestframework>` | >= 3.4        | No             |
+| :ref:`dogpile.cache`                             | >= 0.9        | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`dogpile.cache`                             | >= 0.6        | Yes            |
+| :ref:`elasticsearch`                             | >= 1.10, < 8.0 | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`elasticsearch`                             | >= 1.6, < 8.0 | Yes            | 
+| :ref:`falcon`                                    | >= 3.0        | No             |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`falcon`                                    | >= 1.0        | No             |
+| :ref:`fastapi`                                   | >= 0.64       | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`fastapi`                                   | >= 0.51       | Yes            |
+| :ref:`flask`                                     | >= 1.0        | Yes            |
```

Comment:
Is this based off of when Flask dropped support for Python 2.7? Wasn't that with Flask 2.0? https://auth0.com/blog/whats-new-in-flask-2/ Or is it something else?